src/components/Timeline/src/types.ts

```diff
@@ -0,0 +1,23 @@
+import BaseProps from '@gemeente-denhaag/baseprops';
+
+export interface TimelineComponent extends BaseProps {
```

Comment:
The interface name and file name do not match. Also the interface name is incorrect. I would rename to `IStepTypes` to both indicate an interface and more accurately describe the purpose of the interface.

Reply:
We have never prefixed our interfaces with `I`